airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -886,7 +886,7 @@ def stream_slices(
                 sync_mode=SyncMode.full_refresh, cursor_field=cursor_field, stream_slice=stream_slice, stream_state=stream_state
             )
             for record in parent_records:
-                yield {"repository": record["repository"], "project_id": str(record["id"])}
+                yield {"repository": record["repository"], "project_id": record["id"]}
```

Comment:
Is there any reason we change str(..) to just as is it from record[...] ?

Reply:
we need this "project_id" in 2 places:
1) we put it inside state object (as key)
2) we put it inside record object (as value)

for (1) It need to be "string" type
for (2)  I preserve original "integer" type